### ISSUE TITLE
invalid-reverse-solidus error definition has character mixed up.

### ISSUE THREAD

**Issue by JamesNJep** [whatwg/url#772]
Under the host passing section https://url.spec.whatwg.org/#host-parsing 

What I think it is trying to say is that the URL scheme uses the solidus character U+002F (/) and not the reverse solidus character U+005C (\), but the characters seem to be in the wrong order in the sentence above the example.

**Comment by annevk** [comment]
Are you talking about https://url.spec.whatwg.org/#invalid-reverse-solidus?

The characters after the scheme are correct, but those are not the ones in error. The ones later on are.

**Comment by JamesNJep** [comment]
Yes was talking about [https://url.spec.whatwg.org/#invalid-reverse-solidus](https://url.spec.whatwg.org/#invalid-reverse-solidus)  thank you for looking at this.

The part I had trouble understanding is the description sentence above the example.
The description text reads 

> "The URL has a special scheme and it uses U+005C (\) instead of U+002F (/)."

The way I have read and interpreted this when first reading this is that the sentence is making a statement, about how the URL should be written to avoid this error, but I think the sentence is actually trying to make a positive statement to describe the error condition.

I've re-read this table several times and I think the other items in the table are easier to understand because they use words or prases like, "exceeds", "improper", "too many", "too few", "too big", "invalid" that make it clear that the statement is describing an error state. 

I think if the sentence was written more like  the desciption for [https://url.spec.whatwg.org/#example-host-missing](https://url.spec.whatwg.org/#example-host-missing)  and used "but" instead of "and" that would make it easier to understood the statment was describing the error condition if someone is jumping directly to that description.
For example if it was written like this 

> "The URL has a special scheme, but has used U+005C (\) in the place of U+002F (/)."

**Comment by annevk** [comment]
I'm not sure how to apply that consistently. Many of these are statements describing the error state in exactly that fashion, e.g.,

> An [IPv4 address](https://url.spec.whatwg.org/#concept-ipv4) ends with a U+002E (.).

Although, maybe we should remove "instead of U+002F (/)" or put that between parenthesis? Nothing else has advice like that so maybe removing it would be best and make it clearer overall.

What do you think?

**Comment by annevk** [comment]
@TRowbotham any thoughts from you on this feedback?

**Comment by TRowbotham** [comment]
Maybe we could word it like "The URL has a [special scheme](https://url.spec.whatwg.org/#special-scheme) and and it is incorrectly using U+005C (\\) instead of U+002F (/)." or "The URL has a [special scheme](https://url.spec.whatwg.org/#special-scheme) and it was followed by a U+005C (\\)." and dropping the "instead of U+002F (/)" as you suggested. I am fine with editing the wording if it is causing confusion.

**Comment by JamesNJep** [comment]
Just looking back at this again, and adding some additional comments if this helps. 

I've noticed that because I'm viewing the standards page with perfers dark mode as my operating system/browser preference the third column with the "Failure" heading doesn't display well. 
in the "Failure" column cells with "Yes" display with a light background with a light font color and the other cells in this column that display dark only have a "." in the cell.

My understanding of this table is that it is providing definitions for the types of errors that could be identified when parsing an input string to produce a url.  
With the "Failure" column it seem like some of the error are ones the parser could attempt to fix instead of failing to parse the string. But I can't find an equivalent example in the example url parsing table [https://url.spec.whatwg.org/#example-url-parsing](https://url.spec.whatwg.org/#example-url-parsing) 

Do you know if there is a Syntax Diagram for something to show what would be allowed characters at each section of the URL?
